src/applications/debt-letters/components/DebtCardsList.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import reverse from 'lodash/reverse';
+import { connect } from 'react-redux';
+import DebtLetterCard from './DebtLetterCard';
+
+const DebtCardsList = ({ debts, isError }) => {
+  const renderAlert = () => (
+    <div
+      className="usa-alert usa-alert-error vads-u-margin-top--0"
+      role="alert"
+    >
+      <div className="usa-alert-body">
+        <h3 className="usa-alert-heading">
+          Information about your current debts is unavailable
+        </h3>
+        <p className="vads-u-font-family--sans">
+          You can't view information ab out your current debts because something
+          went wrong on our end
+        </p>
+        <p>
+          <strong>What you can do</strong>
+        </p>
+        <p className="vads-u-font-family--sans">
+          You're still able to download your debt letters from the list below.
+          If you need help resolving a debt, or you would like to get
+          information about a debt that has been resolved, call the Debt
+          Management Center at{' '}
+          <a href="tel: 800-827-0648" aria-label="800. 8 2 7. 0648.">
```

Comment:
You could use the `<Telephone>` formation react component here, or change the aria label to `8 0 0. 8 2 7. 0 6 4 8."